internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -142,6 +142,52 @@ func TestAccNetworkRSNetworkPeering_basicGCP(t *testing.T) {
 	})
 }
 
+func TestAccNetworkRSNetworkPeering_updateBasicGCP(t *testing.T) {
+	acc.SkipTestForCI(t) // needs GCP configuration
```

Comment:
is it doable to enable in CI or it's very complicated?

Reply:
I have run the tests but I have done it using my GCP account. If we are able to have a generic account for the team we could do it, but I didn't want to use my account credentials to run the tests in CI.

Reply:
that's for sure.  maybe we can create a triage ticket to have test credentials in GCP

Reply:
I will create the ticket! Thanks